osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -40,16 +40,30 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
 
             double sliderFactor = aimRating > 0 ? aimRatingNoSliders / aimRating : 1;
 
-            if (mods.Any(h => h is OsuModRelax))
```

Comment:
Honestly speed is so convoluted now that I'm not sure if we want to include it or not. I guess it's fine now, but we'll have to reconsider it after splitting speed into flow/tap/rhythm

Reply:
Yeah had the same conversation with apollo. Unless speed is splitted, this is the best option we have.